From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 25 Sep 2016 01:36:13 +0100
Subject: uaccess: Avoid ABI change in 4.7.5
Forwarded: not-needed

Hide the new #include's from genksyms.  I'm not sure whether they'll
change symbol versions but it's a possibility.

---
--- a/arch/mips/include/asm/uaccess.h
+++ b/arch/mips/include/asm/uaccess.h
@@ -14,7 +14,9 @@
 #include <linux/kernel.h>
 #include <linux/errno.h>
 #include <linux/thread_info.h>
+#ifndef __GENKSYMS__
 #include <linux/string.h>
+#endif
 #include <asm/asm-eva.h>
 
 /*
--- a/arch/parisc/include/asm/uaccess.h
+++ b/arch/parisc/include/asm/uaccess.h
@@ -10,7 +10,9 @@
 #include <asm-generic/uaccess-unaligned.h>
 
 #include <linux/bug.h>
+#ifndef __GENKSYMS__
 #include <linux/string.h>
+#endif
 
 #define VERIFY_READ 0
 #define VERIFY_WRITE 1
